servicex_app/tests/resources/transformation/test_get_all.py

```diff
@@ -36,6 +36,14 @@ class TestAllTransformationRequest(ResourceTestBase):
     def example_json():
         return [{'request_id': '123'}, {'request_id': '456'}]
 
+    @staticmethod
```

Comment:
I think this can be a fixture or a method in `ResourceTestBase`

Reply:
The methods have been moved to ResourceTestBase and WebTestBase